frankenphp.go

```diff
@@ -500,6 +500,14 @@ func go_read_post(rh C.uintptr_t, cBuf *C.char, countBytes C.size_t) C.size_t {
 	return C.size_t(readBytes)
 }
 
+//export go_flush_response
+func go_flush_response(rh C.uintptr_t) {
+	r := cgo.Handle(rh).Value().(*http.Request)
+	fc := r.Context().Value(contextKey).(*FrankenPHPContext)
+	fc.responseWriter.(http.Flusher).Flush()
+	println("flushed")
```

Comment:
This is here for now, for testing.

Future @withinboredom: remove this before merging.